README.md

```diff
@@ -18,7 +18,7 @@ cargo install --locked --git https://github.com/r0gue-io/pop-cli
 
 ### Parachains
 
-Use `pop` to generate a new parachain from a template:
+Use `pop` to start a new Parachain project:
```

Comment:
`create` a new parachain project

Reply:
Suggested by Daan here https://github.com/r0gue-io/pop-cli/pull/107#discussion_r1562785274
Changing to create but both `start` and `create` looks better than `generate` for me.